embulk-input-oracle/src/test/java/org/embulk/input/oracle/OracleTests.java

```diff
@@ -33,8 +32,9 @@ public static void execute(TestingEmbulk embulk, String sql) throws IOException
         String user = config.get(String.class, "user");
         String password = config.get(String.class, "password");
         String database = config.get(String.class, "database");
+
         ProcessBuilder pb = new ProcessBuilder(
-                "SQLPLUS",
+                "sql",
```

Comment:
I'm proposing that one should switch to SQLcl (SQL Developer Command Line) (http://www.oracle.com/technetwork/developer-tools/sqlcl/overview/index.html), which is the new sqlplus for this century;) + its backwards compatible with sqplus scripts/commandline options.